meerk40t/main.py

```diff
@@ -123,179 +123,182 @@ def run():
     """
     try:
         from . import kernelserver
-
-        kernel.add_plugin(kernelserver.plugin)
-    except ImportError:
+    except Mk40tImportAbort:
         pass
+    else:
+        kernel.add_plugin(kernelserver.plugin)
 
```

Comment:
kernelsever plugin will *never* raise MK40tImportAbort that will simply never happens

This looks sort of more polite but if the newer paradigm is crash-harder we might as well just move this out of the try altogether.

Reply:
You said that you didn't want main.py to know anything about whether plugins are optional (and abort when their prerequisite packages are missing). I have no issues with removing the try/except from all but those frozen plugins known to need it.

Reply:
The new paradigm is "Don't hide genuine error conditions, but instead allow the exceptions to 'crash' MK - only hide highly specific exceptions that are known to be OK."